package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "changelogplease",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

Comment:
Please don't change versions. That's the maintainer's job and is often scripted as part of a release process.